### DISCUSSION TITLE
intelliSense provided by clangd is inconsistent with the .clang-format settings

### DISCUSSION THREAD

**Discussion by WSFcloud** [clangd/clangd#1436] [Q&A]
When I select the given option，the format of the completed content is not the same as the setting in  .clang-format.
This is written in the .clang-format:
BraceWrapping:
  AfterControlStatement: true
It should wrap after the statement, but it doesn't.
![20221223220231](https://user-images.githubusercontent.com/20496842/209348287-64bb6e8a-d5e9-4e24-a815-1e25ec07ed57.png)
![20221223220238](https://user-images.githubusercontent.com/20496842/209348331-da4a25ac-0625-4b91-ae18-e3dc10983dd3.png)

**Comment by HighCommander4** [comment]
This is a known issue, we have #1235 on file for it

Suggested workaround in the meantime: if you have format-on-save enabled, then saving the file after accepting the snippet should automatically correct the brace placement
